cylc/uiserver/handlers.py

```diff
@@ -145,9 +150,58 @@ def prepare(self):
         super().prepare()
 
 
+class SubscriptionHandler(websocket.WebSocketHandler):
+    subscription_server = None
+    resolvers = None
+    queue = Queue(100)
+
+    def initialize(self, sub_server, resolvers):
+        self.subscription_server = sub_server
+        self.resolvers = resolvers
+
+    def select_subprotocol(self, subprotocols):
+        return GRAPHQL_WS
+
+    def open(self, *args, **kwargs):
+        IOLoop.current().spawn_callback(self.subscription_server.handle, self,
+                                        self.context)
```

Comment:
@dwsutherland when the Tornado handler is created, it adds a callback to the existing loop. The example from the PR passes just `(self.subscription_server.handle, self)`, but I had to include the `self.context` as well. Then in another part of the code had to extend `execute()` method to retrieve the value passed here and propagate it to the cylc-flow schema :+1: